content/en/APIUseCases/create_asset.md

```diff
@@ -0,0 +1,229 @@
+        "id": "some_id",
+        "name": "Saxophone - Alto",
+        "token": "YOUR-V2-ORGANIZATION-TOKEN"
+      },
+      "links": {
+        "ui": {
+          "url": "https://api.cobalt.io/links/not_a_jwt_token"
+        }
+      }
+    }
+  ]
+}
+```
+{{% /expand %}}
+  
+From the output, save the value for `token` as your organization token.
+In our API documentation, you'll see this as `YOUR-V2-ORGANIZATION-TOKEN`.
+
+For more information, see our API reference documentation on the
+[organizations](https://docs.cobalt.io/v2/#organizations) `orgs` endpoint.
+
+## Create an Asset
+
+Now that you have the following information:
+
+- `YOUR-PERSONAL-API-TOKEN`
+- `YOUR-V2-ORGANIZATION-TOKEN` 
+
+You can create an [asset](../getting-started/glossary/#asset) with the following REST call:
+
+```
+curl -X POST "https://api.cobalt.io/assets" \
+  -H 'Accept: application/vnd.cobalt.v2+json' \
+  -H 'Authorization: Bearer YOUR-PERSONAL-API-TOKEN' \
+  -H 'Content-Type: application/vnd.cobalt.v2+json' \
+  -H 'Idempotency-Key: A-UNIQUE-IDENTIFIER-TO-PREVENT-UNINTENTIONAL-DUPLICATION' \
+  -H 'X-Org-Token: YOUR-V2-ORGANIZATION-TOKEN' \
+  --data '{
+            "title": "Test Asset",
+            "description": "How to describe the asset to our pentesters",
+            "asset_type": "web"
+          }' \
+  -v
+```
+
+For more information on each parameter, see our API Reference documentation on
+how to [Create an Asset](https://docs.cobalt.io/v2/#create-an-asset).
+
+The command we use includes a `-v`, which sets up output in verbose mode. The
+command works without it; however, you would see no response from the REST call.
+
+When you review the output of the REST call with the `-v`, look for the line
+with `HTTP/2`. You'll see one of the following lines:
+<!-- The output is associated with a `201` message, which doesn't include
+results, which is why I recommend a `-v` -->
+
+| Message    | Meaning          |
+|------------|------------------|
+| HTTP/2 201 | Asset created    |
+| HTTP/2 401 | No asset created. Check the value of your API Token|
+| HTTP/2 404 | No asset created. Check the value of YOUR-V2-ORGANIZATION-TOKEN|
+| HTTP/2 409 | No asset created |
+
+<!-- Maybe this table really belongs in our API reference, next to
+https://docs.cobalt.io/v2/#errors?  -->
+
+## Find Your Asset ID
+
+To add or modify or information related to your asset, you'll need the asset ID.
+You can find this ID with the REST call to [Get All Assets](https://docs.cobalt.io/v2/#get-all-assets):
+
+```
+curl -X GET "https://api.cobalt.io/assets" \
+  -H "Accept: application/vnd.cobalt.v2+json" \
+  -H "Authorization: Bearer YOUR-PERSONAL-API-TOKEN" \
+  -H "X-Org-Token: YOUR-V2-ORGANIZATION-TOKEN" \
+  | jq .
+```
+
+If you've set up more than one Asset, you may need to search through the output.
+For more information about each Asset response field, see our API documentation
+reference to [Get All Assets](https://docs.cobalt.io/v2/#get-all-assets).
+
```

Comment:
If your organization has more than 10 assets, you can add the `limit` parameter to the request URL to retrieve more entries.

Reply:
I might set that up as a separate note